x-pack/plugins/apm/public/components/fleet_integration/apm_agents/runtime_attachment/runtime_attachment.tsx

```diff
@@ -132,18 +130,36 @@ export function RuntimeAttachment({
                 'xpack.apm.fleetIntegration.apmAgent.runtimeAttachment.version.invalid',
                 { defaultMessage: 'Invalid version' }
               )}
+              helpText={
+                <FormattedMessage
+                  id="xpack.apm.fleetIntegration.apmAgent.runtimeAttachment.version.helpText"
+                  defaultMessage="Enter the {versionLink} of the Elastic APM Java agent that should be attached."
+                  values={{
+                    versionLink: (
+                      <EuiLink
+                        href="https://www.elastic.co/guide/en/apm/agent/java/current/release-notes.html"
```

Comment:
If having something like docLinks is not straightforward for agent docs, can we have something like `{config.docs.base_url}guide...` as in the tutorial?

Reply:
I use docLinks, tkx